app/javascript/packs/pages/HowItWorksPage.jsx

```diff
@@ -0,0 +1,232 @@
+import DoctorWithDevice from "../../packs/components/illustrations/DoctorWithDevice";
+import Volunteer from "../../packs/components/illustrations/Volunteer";
+import Email from "../../packs/components/illustrations/Email";
+import RequestCompleted from "../../packs/components/illustrations/RequestCompleted";
+import Expiration from "../../packs/components/illustrations/Expiration";
+import Disclaimer from "../components/Disclaimer";
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    margin: theme.spacing(3),
+  },
+  videoContainer: {
+    textAlign: "center",
+    marginBottom: theme.spacing(6),
+  },
+  introCopy: {
+    marginBottom: theme.spacing(2),
+  },
+  stepContainer: {
+    marginTop: theme.spacing(1),
+  },
+  step: {
+    padding: theme.spacing(4),
+  },
+  stepHeader: {
+    marginBottom: theme.spacing(1),
+  },
+  stepDetails: {
+    marginTop: theme.spacing(1),
+    padding: theme.spacing(1, 0),
+  },
+  circleContainer: {
+    paddingBottom: theme.spacing(2),
+  },
+  circle: {
+    backgroundColor: theme.palette.primary.main,
+    color: "white",
+    textAlign: "center",
+    width: 50,
+    height: 50,
+    borderRadius: "50%",
+  },
+  number: {
+    fontSize: "1.6rem",
+    fontWeight: "bold",
+    lineHeight: "50px",
+  },
+}));
+
+const HowItWorks = () => {
+  const classes = useStyles();
+
+  const renderStep = (number, header, content, customContent) => {
+    let detailsContent =
+      content.length < 1
+        ? customContent
+        : content.map((copy, i) => <p key={i}>{copy}</p>);
+    return (
+      <Container className={classes.step}>
+        <Container align="center" className={classes.circleContainer}>
+          <div className={clsx(classes.circle)}>
+            <Typography className={classes.number} variant="body1">
+              {number}
+            </Typography>
+          </div>
+        </Container>
+        <Typography align="center" variant="h3" className={classes.stepHeader}>
+          {header}
+        </Typography>
+        <Typography
+          align="center"
+          variant="body1"
+          component="div"
+          className={classes.stepDetails}
+        >
+          {detailsContent}
+        </Typography>
+      </Container>
+    );
+  };
+
+  let stepOneCustomContent = (
+    <>
+      <p>
+        for help with meals, child care, pet care, errands and more on Hospital
+        Hero. <b>Share the link</b> to your request on social media so your
+        friends can respond!
+      </p>
+      <p>
+        We will also <b>publish your request on Hospital Hero</b> so volunteers
+        in your community can find it. (Don't worry, your contact information
+        stays private.)
+      </p>
+      <Typography component="p" variant="caption" align="center">
+        This includes nurses, doctors, EMTs, admins, support staff, cleaning
+        staff, and anyone else who is helping during this crisis.
+      </Typography>
+    </>
+  );
+  let content = {
```

Comment:
Very minor: This can be a constant, I'm indifferent as to whether or not you make the change :)

Reply:
totally agree, updated!